pkg/bisect/bisect_test.go

```diff
@@ -457,6 +466,16 @@ var bisectionTests = []BisectionTest{
 		fixCommit:   "500",
 		isRelease:   true,
 	},
+	// Tests that we do not confuse revisions where the bug was not yet introduced and where it's fixed.
+	// See #4117.
```

Comment:
Maybe add a little more context here that when bisecting into the merge it might determine 790/791(?) and why.

Reply:
Elaborated on the problem.